compiler/il/AliasSetInterface.hpp

```diff
@@ -266,7 +260,7 @@ class TR_SymAliasSetInterface : public TR_AliasSetInterface<TR_SymAliasSetInterf
    setAliases_impl(TR::SparseBitVector &aliasesToModify, bool value, bool isDirectCall, bool includeGCSafePoint)
       {
       TR::Compilation *comp = TR::comp();
-      TR_ASSERT(_aliasSetType == useDefAliasSet || _aliasSetType ==  UseOnlyAliasSet, "failed: can only call setAliases_impl for UseOnly or useDef presently");
+      TR_ASSERT(_AliasSetInterface == useDefAliasSet || _AliasSetInterface ==  UseOnlyAliasSet, "failed: can only call setAliases_impl for UseOnly or useDef presently");
```

Comment:
should be TR_ASSERT_FATAL IMO to make sure this is checked...

Reply:
Have changed to `TR_ASSERT_FATAL` in 091c9f0